thumbhash.cpp

```diff
@@ -16,9 +16,6 @@ struct thumbhash_encoder_struct {
 thumbhash_encoder thumbhash_encoder_create(void* buf, size_t buf_len)
 {
     thumbhash_encoder e = new struct thumbhash_encoder_struct();
```

Comment:
@skidder wouldn't it be better to enclose this into a try-catch block? If the `std::bad_alloc` is encountered, we should catch it and return `NULL` since the function is already designed in a way that it returns `NULL` when `new` fails